spec/requests/items_spec.rb

```diff
@@ -95,39 +95,78 @@
   describe "POST api/v1/items" do
     context "when item post request attributes are valid" do
       it "creates a new item in the database" do
+        credentials_manager = Api::V1::CredentialsManager.new
```

Comment:
Recommendation for Refactoring: Same here - you can probably extract the duplication into a `before_each` block